docs/marionette.regionmanager.md

```diff
@@ -103,7 +103,7 @@ var rm = new Marionette.RegionManager();
 var regions = rm.addRegions({
   main: '#main-content',
   navigation: {
-    selector: '#navigation',
```

Comment:
My thinking was to maintain links for a short time period and direct users to the new docs until we've got something better for regions.